src/Extension/FluentExtension.php

```diff
@@ -1021,7 +1021,11 @@ protected function requireSavedInLocale()
         }
 
         if ($this->owner->config()->get('cms_publish_required') !== null) {
-            Deprecation::notice('5.0', 'Use cms_localisation_required instead');
+            Deprecation::notice(
+                '5.0.0',
+                'The "' . self::class . '.cms_publish_required" configuration option is deprecated. Use "'
+                    . self::class . '.cms_localisation_required instead'
+            );
```

Comment:
The original message implies we should use something called `cms_localisation_required` instead of the `requireSavedInLocale()` method, which is misleading.

The correct way to deprecate config is to add a `@deprecated` annotation to the phpdoc for the `private static` config variable... but since this module doesn't declare those, this will probably be fine.